app/views/profile.html

```diff
@@ -81,13 +84,15 @@
               <div class="form-group">
                 <label>Reports</label>
                 <div class="checkbox" style="margin-top: 0px;">
-                  <label tooltip="Receive Daily Channel Reports">
+                  <label>
                     <input type="checkbox" ng-model="user.dailyReport" /> Receive Daily Channel Reports?
+                    <i ng-show="uiSettings.showTooltips" tooltip-placement="top" tooltip="Receive Daily Channel Reports" class="glyphicon glyphicon-question-sign tooltip-wide"></i>
```

Comment:
Maybe make the tooltip read: 'Receive daily channel reports that give a summary of the transactions run through channel that you can view.'